pygmt/figure.py

```diff
@@ -298,6 +302,10 @@ def show(self, dpi=300, width=500, method=None):
             - **external**: PDF preview in an external program [default]
             - **notebook**: PNG preview [default in Jupyter notebooks]
             - **none**: Disable image preview
+        waiting : float
+            Suspend the execution of the current process for a given number of
+            seconds after launching an external viewer.
+            Only works if ``method="external"``.
```

Comment:
```suggestion
            seconds after launching an external viewer. [Default is 0.5].
            Only works if ``method="external"``.
```

Reply:
It's clear that `waiting` has a default value of 0.5 from the function parameter `waiting=0.5`, thus no need to mention it in the docstrings.